Client/mods/deathmatch/logic/luadefs/CLuaEffectDefs.cpp

```diff
@@ -498,6 +499,58 @@ int CLuaEffectDefs::fxAddFootSplash(lua_State* luaVM)
     return 1;
 }
 
+int CLuaEffectDefs::FxAddShadow(lua_State* luaVM)
+{
+    CScriptArgReader argStream(luaVM);
+    eShadowType      shadowType = SHADOW_NONE;
```

Comment:
API design looks good although I would favour "default", "none", or "plain" as a shadowType, instead of it being optional.